src/state/slices/opportunitiesSlice/selectors/combined.ts

```diff
@@ -96,17 +97,56 @@ export const selectAggregatedEarnOpportunitiesByAssetId = createDeepEqualOutputS
     const combined = [...userStakingOpportunites, ...userLpOpportunities]
     const totalFiatAmountByAssetId: Record<AssetId, BN> = {}
     const projectedAnnualizedYieldByAssetId: Record<AssetId, BN> = {}
+
+    const isActiveOpportunityByAssetId = combined.reduce<Record<AssetId, boolean>>((acc, cur) => {
```

Comment:
Two hard things in software engineering, this is a record of AssetId to active opportunity in the context of APYs:

- either an earn page AssetId balance without earn/rewards filter applied, but with a positive fiat value nevertheless since it is what makes an opportunity active
- or an earn balance tab AssetId balance with a positive fiat value
- or a rewards balance tab AssetId balance with a positive fiat value